transfer-repo-into-the-org.md

```diff
@@ -0,0 +1,75 @@
+# Transferring an existing repository into the organization
+
+## Step 1. Get the repository ready
+
+Ideally, the repository should have the following documents in place:
+
+- `CODE_OF_CONDUCT.md`: it can be a reference to
+  [the Node.js Code of Conduct][coc].
+- `CONTRIBUTING.md`: if there isn't one already, [the contributing guide][]
+  of Node.js core could be a good example. Consider including the Developer's
+  Certificate of Origin section in the document to avoid potential copyright
+  conflicts.
+- `LICENSE`, or other kind of documents that describe the license of
```

Comment:
Another question: what licenses are allowed? AFAICT, MIT is safe, not sure about Apache v2, GPL and other popular licenses.

Reply:
This may be something we'd like to ask the @nodejs/foundation team for legal advice on, in addition to our own ideas and opinions.